dbms/cmake/version.cmake

```diff
@@ -59,6 +59,11 @@ execute_process(
   OUTPUT_STRIP_TRAILING_WHITESPACE
   )
 
+if (FORCE_TIFLASH_RELEASE_VERSION)
+    message (STATUS "force set TIFLASH_RELEASE_VERSION to ${FORCE_TIFLASH_RELEASE_VERSION}")
+    set (TIFLASH_RELEASE_VERSION ${FORCE_TIFLASH_RELEASE_VERSION})
+endif()
+
```

Comment:
what is it used for?

Reply:
We can manually set TiFlash's version rather than deducing from git tags when debugging.

Reply:
when do we actually need to use it?

Reply:
When patching TiFlash

Reply:
I think pd-ctl `config set cluster-version <xxx>` can do better than recompiling tiflash?

Reply:
What if TiFlash's version differs all other components' version?

Reply:
Will remove that